packages/core/src/analytics/integrations/__tests__/Omnitracking.test.js

```diff
@@ -352,9 +353,10 @@ describe('Omnitracking', () => {
     });
     describe('Should send track events when', () => {
       it('event is `Place Order Started`', async () => {
-        const tid = 10097;
+        const tid1 = 10097;
```

Comment:
Nitpicking: Is it possible to give more descriptive names for those tids? It would be easier if they have the corresponding action name to identify what they mean. If it is not possible, you can disregard this comment.

Reply:
Fixed on dev-main-22-02-22. thanks 🙂